### ISSUE TITLE
Completion issue on readline shell with CASE_SENSITIVE_COMPLETIONS

### ISSUE THREAD

**Issue by mitnk** [xonsh/xonsh#1342]
When using readline and set `$CASE_SENSITIVE_COMPLETIONS = False`.

```
$ mkdir usr && mkdir Users
$ cd ./u<TAB>
```

The `u` in second cmd will be eaten. No such issue with PTK.

**Comment by adqm** [comment]
Ah, okay.  I can see why this is happening.  I may be able to put in a fix today.

**Comment by mitnk** [comment]
@adqm great! I spent some time but didn't get much.

However, If I put the following line in `~/.inputrc`:
 
```
set completion-ignore-case on
```

So it seems a consistent issue between `xonsh` and `readline` :)

**Comment by mitnk** [comment]
@adqm I'm trying fix it in #1349 :)
